content/docs/concepts/architecture/index.md

```diff
@@ -125,10 +129,13 @@ The default Keptn installation includes Keptn-services for some Execution Plane
   about whether a task sequence (such as the artifact delivery) should continue or not.
   If the approval strategy within a stage has been set to `manual`,
   the gatekeeper service does not respond with any event since, in that case,
-  the user is responsible for sending an `approval.finished` event (either via the Bridge or via the API).  
+  the user is responsible for sending an `approval.finished` event
+  (using either the [Keptn Bridge](../../0.19.x/bridge/#approvals) or the API).  
 
 - **remediation-service:** determines the action to be performed in remediation workflows. 
 
+- **mongodb-datastore:** stores MongoDB event data that is deployed in the cluster.
```

Comment:
I think this section can be removed, as the mongodb-datastore has already been mentioned above